generate-language-platform-bundles/generate_language_platform_bundle.sh

```diff
@@ -43,8 +43,13 @@ mkdir -p  "${RUNNER_DIR}/record/bin"
 ${SCRIPT_CURRENT_DIR}/download.sh "${REMOTE_RECORDER_JAR}" "${RUNNER_DIR}/record/bin/record-and-upload.jar"
 
 # 4. Place Recording script
-LOCAL_RECORDER_SCRIPT="${SCRIPT_CURRENT_DIR}/record/record_screen_and_upload.sh"
-cp "${LOCAL_RECORDER_SCRIPT}" "${RUNNER_DIR}/record_screen_and_upload.sh"
+FILE_EXT="sh"
```

Comment:
Do you want to provide both the `.sh` and `.bat`  files for all platforms irrespective of it is Linux/MacOS or Windows

Reply:
No, only .bat for Windows and .sh for Linux/Macos